src/Raven.Server/ServerWide/ClusterStateMachine.License.cs

```diff
@@ -29,24 +31,33 @@ public sealed partial class ClusterStateMachine
 {
     private const int MinBuildVersion60000 = 60_000;
     private const int MinBuildVersion60102 = 60_102;
```

Comment:
This is incorrect. 6.0.102 build number is 60026:
![image](https://github.com/user-attachments/assets/776a2142-e109-4ebf-96a2-1a76c8923e59)

Reply:
Done